extra/1-currency-conversion.js

```diff
@@ -15,7 +18,13 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+
+function convertToBRL(PoundExchange) {
```

Comment:
Hi Bedrije

what you did here is was wonderful and also  you can make it short like

function convertToBRL(price) {
  let priceConverted = price * 0.99 * 5.7
  return parseFloat(priceConverted .toFixed(2))
}

Thanks

Reply:
Thank you for your feedback Abubakar